### PR TITLE
Fix undefined "event" error

### PR DESCRIPTION
**Pull request by artsjedi**
"event" was undefined in this scope,  fixed to the correct variable name.
refer to this issue #3668

### CODE REVIEW

**Comment by artsjedi** [comment]
refer to this issue #3668

File: src/interaction/InteractionManager.js [hunk 1]
```diff
@@ -1430,7 +1430,7 @@ export default class InteractionManager extends EventEmitter
         // This is the way InteractionManager processed touch events before the refactoring, so I've kept
         // it here. But it doesn't make that much sense to me, since mapPositionToPoint already factors
         // in this.resolution, so this just divides by this.resolution twice for touch events...
-        if (navigator.isCocoonJS && event.pointerType === 'touch')
+        if (navigator.isCocoonJS && interactionEvent.pointerType === 'touch')
```
Comment (andrewstart) [inline src/interaction/InteractionManager.js]: It actually should be `pointerEvent` - that is the one with a `pointerType` property.

Comment (artsjedi) [inline src/interaction/InteractionManager.js]: ok, I will change, But both of them works on cocoon.

**Comment by andrewstart** [comment]
Oh, this is targeting my bugfix branch, which I had completely removed the commits that included logs from. I'll just add this to the branch directly.

**Pull request closed without merging**